README.md

```diff
@@ -90,12 +90,74 @@ When you have addressed all technical review and peer review comments, notify th
 
 The developers check that the **Tech review passed** label has been added to the pull request and peer pull request approval provided, then accept it.
 
-## Installing Pantheon
+## Deploying Pantheon
 
-### Prerequisites
+Install the Pantheon codebase, perform some basic initial configuration, and deploy Pantheon with a containerized MongoDB database.
+
+**Prerequisites**
+* Tested on Red Hat Enterprise Linux (RHEL) 8.3 and Fedora 33.
+* openJDK
 * Podman
-* Buildah
-* Java
+
+**Procedure**
+
+1. Clone the git repo:
+
+        $ git clone https://github.com/redhataccess/pantheon.git
+
+2. Set the `PANTHEON_CODEBASE` environment variable to the root directory of the repo:
+
+        $ export PANTHEON_CODEBASE=<path>/pantheon/
+
+3. Set your `JAVA_HOME` variable:
+
+
+        $ vi ~/.bashrc
+        ...
+        # User specific aliases and functions
+        export JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:bin/java::")
+
+
+4. Start a mongo container on port `27017`:
+
+        $ podman run -p 27017:27017 --name mongodb -d mongo
+
+5. Run the `deploy_local.sh` script:
+
+        $ sh scripts/deploy_local.sh
+
+6. Navigate to http://localhost:8181/pantheon to view the Pantheon UI. Log in to the dashboard with username `admin` and password `admin`
+
+## Uploading content to Pantheon with `pantheon-uploader`
+
+Use the `pantheon-uploader` tool to upload local content to Pantheon.
+
+1. Install the `pantheon-uploader` tool:
+
+        $ wget -S https://raw.githubusercontent.com/redhataccess/pantheon/master/uploader/setup.sh
+        $ sh setup.sh
```

Comment:
we recenty retired this uploader. The new uploader has its own repository. Please see readme here:
https://github.com/redhataccess/pantheon-uploader

With the new uploader, you can do pip install

Reply:
Awesome, thanks

Reply:
Installing using pip, for example pip-3:

pip-3 install --no-cache-dir git+https://github.com/redhataccess/pantheon-uploader.git